lib/slide_hub/cloud/queue/response.rb

```diff
@@ -0,0 +1,33 @@
+module SlideHub
+  module Cloud
+    module Queue
+      class Message
+        attr_accessor :id
+        attr_accessor :body
```

Comment:
__[reek]__
* [Attribute] is a writable attribute [:ref](https://github.com/troessner/reek/blob/master/docs/Attribute.md)